docs/docs-content/palette-cli/commands/pde.md

```diff
@@ -0,0 +1,422 @@
+  | `--help` | Help for the `delete` subcommand. | - |
+
+
+### Describe
+
+Use the `describe` subcommand to describe an app profile. To describe an app profile, you must provide the name and version of the app profile. The `describe` subcommand supports the following flags.
+
+  | **Flag**              | **Description**   | **Type**    |
+  |-------------------|-----------------------|---------|
+  | `--name` | The name of the app profile. | string |
+  | `--version` | The version of the app profile. | string |
+  | `--help` | Help for the `describe` subcommand. | - |
+
+
+### List
+
+Use the `list` subcommand to list all app profiles. The `list` subcommand supports the following flags.
+
+  | **Flag**              | **Description**   | **Type**    |
+  |-------------------|-----------------------|---------|
+  | `--name-filter` | Filter the list of app profiles by providing a name. | string |
+  | `--help` | Help for the `list` subcommand. | - |
+
+
+### Versions
+
+Use the `versions` subcommand to list all versions of an app profile. To list all versions of an app profile, you must provide the name of the app profile. The `versions` subcommand supports the following flags.
+
+  | **Flag**              | **Description**   | **Type**    |
+  |-------------------|-----------------------|---------|
+  | `--name` | The name of the app profile. | string |
+  | `--help` | Help for the `versions` subcommand. | - |
+
+## Cluster Group
+
+Use the `cluster-group` command to change the cluster group that commands will target. You can also list all available clusters. The `cluster-group` supports the following subcommands.
+
+  * `list`   -      List Palette Cluster Groups.
+
+  * `switch` -      Switch your active Palette Cluster Group.
+
+
+  ### List
+
+  Use the `list` subcommand to list all Palette Cluster Groups. No additional flags are required.
+
+  ### Switch
+
+  Use the `switch` subcommand to switch your active cluster group. The `switch` subcommand supports the following flags.
+
+  | **Flag**              | **Description**   | **Type**    |
+  |-------------------|-----------------------|---------|
+  | `--name` | The name of the cluster group. | string |
+  | `--scope`| The scope of the cluster group. Allowed values are `project`, `tenant`, and `system`. | string |
+  | `--help` | Help for the `switch` subcommand. | - |
+
+
+## Registry
+
+Use the `registry` subcommand to manage Palette Registries for App Mode. The `registry` subcommand supports the following subcommands.
+
+  * `create` - Create a Palette Registry.
+
+  * `delete` - Delete a Palette Registry.
+
+  * `list`   - List Palette Registries. 
+
+
+  ### Create
+
+  The `create` subcommand creates a Palette Registry. You can specify an Open Container Initiative (OCI) or a Helm registry. Use the subcommand `oci` or `helm` to specify the type of registry you want to create. The `create` subcommand supports the following subcommands. 
```

Comment:
Does Open Container Initiative (OCI) need to modify anything here? Such as "...Open Container Initiative (OCI) _registry_ or a Helm registry."

Italics aren't intended for the doc - it's only intended to highlight the added word. :)